Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/model/Tasks.java

```diff
@@ -85,4 +94,17 @@ public boolean apply(Map.Entry<Id, SyncedData<Task>> input) {
             }
         };
     }
+
+    public boolean hasSyncError() {
+        return !internalMap().filter(syncError()).isEmpty();
+    }
+
+    private static Predicate<Map.Entry<Id, SyncedData<Task>>> syncError() {
```

Comment:
Fixed in https://github.com/novoda/spikes/pull/113/commits/3a3ea23f20b9d724c6cc0b4a3a95287177262a07